app/controllers/posts_controller.rb

```diff
@@ -25,6 +25,12 @@ def create
     end
   end
 
+  def destroy
+    @post = Post.find(params[:id])
+    @post.destroy
+    redirect_to user_path(@post.author), notice: 'Post deleted successfully'
+  end
+
```

Comment:
- [ ] I see that you implemented the `destroy` method in your controllers, however, you haven't added the authorization to your actions! 😱 Please consider taking a look at this article from [CanCanCan official docs](https://github.com/CanCanCommunity/cancancan/blob/develop/docs/controller_helpers.md#authorize_resource-load_resource-load_and_authorize_resource) on loading and authorizing resources and implement this! 🤓 🔧

Reply:
added for both, thanks for the observation!